README.md

```diff
@@ -5,7 +5,7 @@
   </a>
 </p>
 
-[![Join the chat at https://gitter.im/scala-js/scala-js](https://badges.gitter.im/scala-js/scala-js.svg)](https://gitter.im/scala-js/scala-js)
+Chat: [#scala-js](https://discord.com/channels/632150470000902164/635668814956068864) on Discord.
```

Comment:
Does that link work for someone who hasn't registered to the server yet? I think that publicly we must always use the invite link https://discord.com/invite/scala

Reply:
Ah... I didn't know that. To bad we can't offer a deeplink...